configor_test.go

```diff
@@ -6,12 +6,12 @@ import (
 	"io/ioutil"
 	"os"
 	"reflect"
+	"regexp"
 	"testing"
 
-	"gopkg.in/yaml.v2"
-
 	"github.com/BurntSushi/toml"
-	"github.com/jinzhu/configor"
+	"github.com/minitauros/configor"
```

Comment:
@minitauros this change breaks the tests.

Reply:
Thing is, if I import the original package there, the test will fail, because the code under test is in my package and not in the original one. What to do here?

Reply:
You can create a new branch to complete the pull request, and this will not influence your master branch in your own repository.